src/intTest/java/uk/nhs/digital/nhsconnect/nhais/uat/CustomArgumentsProvider.java

```diff
@@ -43,7 +45,7 @@ public CustomArgumentsProvider(String folder) {
             .peek(es -> {
```

Comment:
same here, not sure on the purpose of es

Reply:
abbreviations of parameter names to lambdas / streams are usually considered ok